### ISSUE TITLE
Batch action not working except when using the confirm option

### ISSUE THREAD

**Issue by Uelb** [activeadmin/activeadmin#8403]
I'm using the latest [v4beta7 release ](https://github.com/activeadmin/activeadmin/releases/tag/v4.0.0.beta7)
I'm trying  to implement a batch action for one of my resource like this : 
````ruby
  batch_action :publish do |ids|
    batch_action_collection.find(ids).each(&:publish!)
    redirect_to collection_path
  end
````

However, when I click on the batch action button, nothing happens for the user, all I can see is a modification of the DOM, the input batch_action input of the `#collection_selection` form has a different value : 
````html
<input name="batch_action" id="batch_action" type="hidden" value="unpublish">
````

When I try again with a confirm message like so :
````ruby
  batch_action :publish, confirm: 'test' do |ids|
    batch_action_collection.find(ids).each(&:publish!)
    redirect_to collection_path
  end
````

Everything works as inteded, I have the confirm dialog, and the action is triggered when I confirm.

### Expected behavior

The batch action should trigger even when we do not specify a confirm option

### Actual behavior

The batch action does not trigger, and nothing happens from the user point of view.

### How to reproduce

Here is a minimal resource to try

````ruby
ActiveAdmin.register Post do
  batch_action :publish do |ids|
    batch_action_collection.find(ids).each(&:touch) # I changed it to touch here so it's easier to see
    redirect_to collection_path, alert: 'This should work'
  end
end
````

**Comment by javierjulio** [comment]
@Uelb this has already been resolved by #8389 I just need to cut a release with that change. I'll get to that soon. Thank you.

**Comment by Uelb** [comment]
Thank you for the answer.
